checks/script_tag.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Check if:
+ * A <script> tag is included in header.php or footer.php
+ */
+class Script_Tag implements themecheck {
+	protected $error = array();
+
+		function check( $php_files, $css_files, $other_files ) {
+
+		$ret = true;
+
+		checkcount();
+		/**
+		 * This check is limited to header.php and footer.php.
+		 */
+		foreach ( $php_files as $file_path => $file_content ) {
+			$filename = tc_filename( $file_path );
+			if ( 'footer.php' === $filename && preg_match( '/<script/', $file_content ) || 'header.php' === $filename && preg_match( '/<script/', $file_content )) {
```

Comment:
```suggestion
			if ( ! in_array( $filename, array( 'header.php', 'footer.php' ) ) ) {
				continue;
			}

			if ( preg_match( '/<script/i', $file_content ) ) {
```

Seems much easier to read in this case, and checking for `<script` case insensitively.